### ISSUE TITLE
signals for farms

### ISSUE THREAD

**Issue by Darvame** [unbit/uwsgi#1024]
Does it work?

**Comment by xrmx** [comment]
Hi there, isn't this more of a question than an issue? If so better discuss on mailing list no?

**Comment by Darvame** [comment]
it looks like uwsgi_signal_wait() does not see 'farm' recievers

**Comment by Darvame** [comment]
naaah

**Comment by unbit** [comment]
@Darvame yes mule farming is incomplete (basically because they are not very common albeit they open to very interesting scenarios). Pull requests are well-comed :)

**Comment by unbit** [comment]
@Darvame btw the function uwsgi_route_signal() is what manages signal routing and supports (in theory) farming:

https://github.com/unbit/uwsgi/blob/master/core/signal.c#L388
